Functions.Templates/Templates/SqlInputBinding-Python/__init__.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import azure.functions as func
+import json
+
+def main(req: func.HttpRequest, rowList: func.SqlRowList) -> func.HttpResponse:
```

Comment:
`// Visit https://aka.ms/sqlbindingsinput to learn how to use this input binding`